middleware/src/hsm/glue.go

```diff
@@ -0,0 +1,46 @@
+package hsm
+
+import (
+	"log"
+
+	"github.com/digitalbitbox/bitbox02-api-go/communication/usart"
+	"github.com/flynn/noise"
+)
+
+// See ConfigInterace: https://github.com/digitalbitbox/bitbox02-api-go/blob/e8ae46debc009cfc7a64f45ec191de0220f0c401/api/firmware/device.go#L50
+type bitbox02Config struct{}
+
+func (bb02Config *bitbox02Config) ContainsDeviceStaticPubkey(pubkey []byte) bool {
+	return false
+}
+func (bb02Config *bitbox02Config) AddDeviceStaticPubkey(pubkey []byte) error {
+	return nil
+}
+func (bb02Config *bitbox02Config) GetAppNoiseStaticKeypair() *noise.DHKey {
+	return nil
+}
+func (bb02Config *bitbox02Config) SetAppNoiseStaticKeypair(key *noise.DHKey) error {
```

Comment:
Can you rename these to HSM and Middleware?

Reply:
I can't, as it implements the bitbox02-api-go firmware.ConfigInterface.

I could rename the interface functions in a later PR to `Host` instead of `App`, but this is out of scope for this PR.